src/apiv2.ts

```diff
@@ -361,11 +361,14 @@ export class Client {
 
     let body: ResT;
     if (options.responseType === "json") {
-      // 204 statuses have no content. Don't try to `json` it.
-      if (res.status === 204) {
+      const text = await res.text();
```

Comment:
It might be good to get @bkendall 's approval on the `api` changes, even though it's only a few lines

Reply:
Agreed. Will wait on his approval before submitting.